src/renderer/components/atoms/button/Button.stories.mdx

```diff
@@ -0,0 +1,138 @@
+<!-- Button.stories.mdx -->
```

Comment:
This file should be named in downcase. `button.stories.mdx`

Reply:
right, done!